src/components/buttons/MinimalButton.tsx

```diff
@@ -49,14 +50,14 @@ const MinimalButtonComponent = (props: Props) => {
   )
 
   return (
-    <TouchableOpacity disabled={disabled || pending} style={memoizedStyle} onPress={handlePress}>
+    <EdgeTouchableOpacity disabled={disabled || pending} style={memoizedStyle} onPress={handlePress}>
```

Comment:
This guy has also incorporated the `usePendingPress` hook.

Reply:
This caller explicitly sets different opacities depending on the type of "busy" state

Reply:
We can probably blanket apply this styling to any `EdgeTouchableOpacity` but I'd like to wait to decide on that in case we missed something